docs/install/configuration.md

```diff
@@ -112,6 +113,23 @@ The `ncbi_database` entry contains the path to the required local copy of the NC
 database, ([see here](build_local_ncbi.md)). This database **must** be included even
 when no NCBI taxa are included.
 
+### Project IDs
+
+The SAFE data system was initially developed as part of the SAFE project, which
+organises datasets into projects. Each project is identified using a unique
+identification number, and each dataset is associated with one or more projects using
+these project ID numbers. We anticipate that other deployments of the SAFE data system
+may not wish to organise datasets in this manner. If so, the checking of project IDs can
+be turned off by setting the `use_project_ids` option  to `False`. With this option off,
+datasets will **fail validation** if they contain project IDs.
+
+!!! Warning
+
+  Each deployment of the SAFE data system will have to make a choice of whether or not
+  to organise datasets into project. Once this choice has been made it will be binding.
+  As such, this option should only be changed by the relevant data manager during the
+  initial setup of the data system.
+
```

Comment:
I don't think we've got a clear branding for the new system yet. I'm leaning towards `safedata` to try and more explictly break the link back to SAFE. Then we have the `safedata` system with `safedata_validator`, `safedata_server` and the `safedata` R package (which I really wish I'd called `safedatar` or something now, but that ship has sailed). Thoughts?

```suggestion
### Project IDs

The `safedata` system was initially developed as part of the SAFE project, which
organises datasets into projects. Each project is identified using a unique
identification number, and each dataset is associated with one or more projects using
these project ID numbers. We anticipate that other deployments of the `safedata` system
may not wish to organise datasets in this manner. If so, the checking of project IDs can
be turned off by setting the `use_project_ids` option  to `False`. With this option off,
datasets will **fail validation** if they contain project IDs.

!!! Warning

  Each deployment of the `safedata` system will have to make a choice of whether or not
  to organise datasets into project. Once this choice has been made it will be binding.
  As such, this option should only be changed by the relevant data manager during the
  initial setup of the data system.

```

Reply:
Yep I like `safedata` as a name, probably not the end of the world that the R package has the same name as that is the component that the largest number of users will interact with